### PR TITLE
Replaced OC with SFX instrumentation for Go frontend service

### PR DESCRIPTION
**Pull request by owais**

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

File: kubernetes-manifests/frontend.yaml [hunk 1]
```diff
@@ -76,6 +76,10 @@ spec:
                 fieldPath: status.hostIP
           - name: OC_SERVICE_ADDR
             value: "$(NODE_IP):55678"
+          - name: SIGNALFX_ENDPOINT_URL
```
Comment (dmitryax) [inline kubernetes-manifests/frontend.yaml:79]: Is it being used?

**Review by tigrannajaryan — COMMENTED** [review]
I like how minimal codechanges needed for enabling instrumentation are.

File: kubernetes-manifests/frontend.yaml (hunk 1 above)
Comment (owais) [reply kubernetes-manifests/frontend.yaml:79]: Yes. Tracer pick it up automatically.

**Comment by owais** [comment]
> I like how minimal codechanges needed for enabling instrumentation are.

wait till you see the next PR :)

**Pull request merged**